GNUmakefile

```diff
@@ -41,13 +41,9 @@ cov:
 	gocov test ${GOFILES} | gocov-html > /tmp/coverage.html
 	open /tmp/coverage.html
 
-test:
-	@./scripts/verify_no_uuid.sh
-	@env \
-		GOTAGS="${GOTAGS}" \
-		GOFILES="${GOFILES}" \
-		TESTARGS="${TESTARGS}" \
-		sh -c "'$(CURDIR)/scripts/test.sh'"
+test: dev
+	go test -tags "$(GOTAGS)" -i -run '^$$' ./...
```

Comment:
Done. I've also dropped the `TEST ?= ./...` since fast tests remove the need for testing specific packages via `make`. You can always run `go test` in the package itself.